src/v1/creative-auction/types.ts

```diff
@@ -0,0 +1,91 @@
+import { AuctionResult, Banner, CreativeAdItem } from '../types/external';
+
+export type CreativeAuctionParams = {
+  requestId: string;
+  sessionId?: string;
+  user?: {
+    userId?: string;
+    yearOfBirth?: number;
+    gender?: string;
+    interests?: string[];
+  };
+  device?: {
+    os?: string;
+    osVersion?: string;
+    advertisingId?: string;
+    uniqueDeviceId?: string;
+    model?: string;
+    ip?: string;
+  };
+  inventory: {
+    inventoryId: string;
+    numItems: number;
+    items?: string[];
+    categories?: string[];
+    searchQuery?: string;
+    searchMetadata?: {
+      synonyms?: string[];
+    };
+  };
+  pageId?: string;
+};
+
+export type CreativeAuctionHttpRequestBody = {
+  request_id: string;
+  session_id?: string;
+  user?: {
+    user_id?: string;
+    year_of_birth?: number;
+    gender?: string;
+    interests?: string[];
+  };
+  device?: {
+    os?: string;
+    os_version?: string;
+    advertising_id?: string;
+    unique_device_id?: string;
+    model?: string;
+    ip?: string;
+  };
+  inventory: {
+    inventory_id: string;
+    num_items: number;
+    items?: string[];
+    categories?: string[];
+    search_query?: string;
+    search_metadata?: {
+      synonyms?: string[];
+    };
```

Comment:
`CreativeInventory` field is a bit different with `(Item)Inventory` it doesn't have `num_items` and `search_metadata`.

Reply:
Addressed in https://github.com/moloco-rmp/decision-api-client-js/pull/11/commits/77ac89441800b3c1ab8f717320f3af95faebc560
Thanks for the review.